app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -179,6 +196,10 @@ export default {
     },
     // eslint-disable-next-line no-unused-vars
     async onSubmit(submission) {
+      if (this.preview) {
+        alert('Submission disabled during form preview');
```

Comment:
Should consider invoking the notification popup for this instead of an alert.

Reply:
Wouldn't be an error though. I think it could be the base dialog rather than an alert but I need to get Mina's feedback on this so will address in a UX cleanup ticket.